config/application.rb

```diff
@@ -2,11 +2,19 @@
 
 require 'rails'
 
+# Pick the frameworks you want:
 require 'active_model/railtie'
+# require "active_job/railtie"
 require 'active_record/railtie'
-
+# require "active_storage/engine"
 require 'action_controller/railtie'
+# require "action_mailer/railtie"
```

Comment:
Since these components aren't used, can we remove the comments?

Reply:
We could, but if we ever decide to enable any new components, it's easier with them just commented out. I took the style from the rails upgrade tool with comments unused components.